toolchain/sem_ir/name.cpp

```diff
@@ -25,6 +25,8 @@ static auto GetSpecialName(NameId name_id, bool for_ir) -> llvm::StringRef {
       return "package";
     case NameId::Base.index:
       return "base";
+    case NameId::Vptr.index:
+      return "vptr";
```

Comment:
`vptr` isn't a keyword, so this might be misleading when used to print Carbon-like syntax. Maybe:
```suggestion
      return for_ir ? "vptr" : "<vptr>";
```